plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -85,6 +87,12 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 		case configMapResource:
 			return r.authorizeGet(nodeName, configMapVertexType, attrs)
 		case pvcResource:
+			if attrs.GetSubresource() == "status" {
```

Comment:
Check the feature is enabled before checking that the subresource is status.

Reply:
sure, I can do that. but why? sorry I am still bit of newb with this area of k8s code - trying to understand.

Reply:
I'd like the code related to the feature to be entirely enclosed in a code block guarded by the feature flag.

Reply:
fixed